ramls/quantity.json

```diff
@@ -4,9 +4,8 @@
   "type": "object",
   "properties": {
     "quantity": {
-      "description": "Quantity of the number of times the interval repeats",
-      "type": "number",
-      "minimum": 1.0
+      "description": "Recurring fine amount",
```

Comment:
To avoid confusion in the future, it's probably better to emphasize on the relationship between quantity and interval in the description, something like "Fine amount per interval"

Reply:
done